dags/experiments_live.py

```diff
@@ -17,7 +17,7 @@
 
 with DAG('experiments_live',
          default_args=default_args,
-         concurrency=1,
+         concurrency=4,
          max_active_runs=1,
```

Comment:
Based on the [docs](https://airflow.apache.org/docs/apache-airflow/stable/_api/airflow/models/dag/index.html#airflow.models.dag.DAG) this will limit the number of tasks concurrently executed in the DAG to 4 and also prevent more than 1 DAG from running at the same time.